backend/cli/main.py

```diff
@@ -1,7 +1,6 @@
-#!/Usr/bin/env python
+#!/usr/bin/env python
 import comand_line_parser as par
-import db
-
+from backend.cli import db
```

Comment:
probably after we change the import `main.py` has to be run from a certain directory only. Or PYTHONPATH has to be set. I think that is worth mentioning in the README

Reply:
Done